spec/salestation/web/responses/unprocessable_entity_from_schema_errors_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe Salestation::Web::Responses::UnprocessableEntityFromSchemaErrors do
+  subject(:error) { described_class.create(attributes) }
+  let(:attributes) { {
+    errors: errors,
+    hints: hints,
+  } }
+
+  let(:errors) { {} }
+  let(:hints) { {} }
+
+  context 'error with one message' do
+    let(:errors) { {content: ['is missing']} }
+    let(:hints) { {content: ['is missing']} }
+
+    it 'parses error message' do
+      expect(error.message).to eq("'content' is missing")
+    end
+
+    it 'parses error debug message' do
+      expect(error.debug_message).to eq("'content' is missing")
+    end
+  end
+
+  context 'error with multiple message' do
+    let(:errors) { {content: ['is missing', 'is invalid']} }
+    let(:hints) { {content: ['is missing', 'is invalid']} }
+
+    it 'parses error message' do
+      expect(error.message).to eq("'content' is missing and is invalid")
+    end
+
+    it 'parses error debug message' do
+      expect(error.debug_message).to eq("'content' is missing and is invalid")
+    end
+  end
+
+  context 'error with nested errors' do
+    let(:errors) { {message: {'content'=>['is missing']}} }
+    let(:hints) { {message: {'content'=>['is missing']}} }
+
+    it 'parses error message' do
+      expect(error.message).to eq("'message' => { 'content' is missing }")
```

Comment:
it would have to be `"'message'.content is missing and something else"` which looks odd imo. 
instead of `message` the key could also be `'some key with spaces'`